packages/front-end/components/Layout/Layout.tsx

```diff
@@ -27,9 +27,35 @@ const navlinks: SidebarLinkProps[] = [
     name: "Features",
     href: "/features",
     Icon: BsFlag,
-    path: /^features/,
-    beta: false,
-    className: styles.first,
+    path: /^(features|attributes|namespaces|environments|saved-groups)/,
```

Comment:
You might have to move them out of settings, otherwise it will conflict with the regex for the settings section and it won't know which one to highlight.